model.js

```diff
@@ -56,16 +56,54 @@ const fetchAllUsers = () => {
 };
 
 //This function will respond with an array of all the article objects.
-const fetchAllArticles = () => {
+const fetchAllArticles = ({
+  sort_by = "created_at",
+  order = "desc",
+  topic = "",
+}) => {
+  const validSortBy = [
+    "article_id",
+    "title",
+    "topic",
+    "author",
+    "body",
+    "created_at",
+    "votes",
+  ];
+  const validOrder = ["asc", "desc"];
+  const validTopic = ["", "mitch", "cats", "paper"];
```

Comment:
This will need to be more dynamic. In the case where a new article is added with a new topic, this will consider it and invalid sort query. 
Are there any functions that you already have written that return an array of all the topics?

Reply:
aaaaaaah i was so lazy when i did this, cheers Kyle